taxonomies/food/ingredients.txt

```diff
@@ -578,12 +578,6 @@ en: non-gmo sunflower lecithin
 hr: ne-gmo suncokretov lecitin
 it: lecitina di girasole non ogm
 
-# <en:emulsifier
-< en:sunflower lecithin
-fr: émulsifiant lécithine de tournesol
-fi: emulgointiaine auringonkukkalesitiini
-# ingredient/fr:émulsifiant-lécithine-de-tournesol has 100 products in 6 languages @2019-02-22
```

Comment:
why won't those entries appear again ?

I would leave the entry commented with a comment about why we should not add it

Reply:
They should not appear again because I fixed the parsing so that they are recognized, see this new test: https://github.com/openfoodfacts/openfoodfacts-server/pull/11251/files#diff-ab86aa197f60d1dd875cf1f147d97168ac999c9c3de967adaf5b27c41aa4ec09R89